charts/ckan/templates/ckan/deployment.yaml

```diff
@@ -61,15 +61,23 @@ spec:
               mountPath: /config
               readOnly: true
           command: ["/bin/sh", "-c"]
-          args: ["gunicorn --bind 0.0.0.0:5000 wsgi:application --timeout 120"]
+          args: [{{ .Values.ckan.config.command.args }}]
```

Comment:
It's a good idea to pass the list directly rather than pass a string here, so that (if you need to later on) you can manipulate the items rather than it just being an opaque string. (similarly throughout)

Also you shouldn't need the `sh -c` as the command. You could make the command `[gunicorn]` and then the args can be the part you let the user override via Helm values.

Reply:
ok, I'll try that

Reply:
also, in some situations, I like to have a non running container which is just running a tail so that I can exec onto it and debug it as the stack gets hit.